pkg/ccl/backupccl/alter_backup_test.go

```diff
@@ -0,0 +1,211 @@
+// Copyright 2022 The Cockroach Authors.
+//
+// Licensed as a CockroachDB Enterprise file under the Cockroach Community
+// License (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
+
+package backupccl
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/security"
+	"github.com/cockroachdb/cockroach/pkg/sql"
+	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+	"github.com/cockroachdb/cockroach/pkg/util/log"
+	"github.com/stretchr/testify/require"
+)
+
+// getAWSEncryptionOption wraps getAWSKMSURI in single quotes so it can be used in queries as a string
+func getAWSEncryptionOption(
+	t *testing.T, regionEnvVariable string, keyIDEnvVariable string,
+) string {
+	uri, _ := getAWSKMSURI(t, regionEnvVariable, keyIDEnvVariable)
```

Comment:
hmm we're ignoring the error returned here, what we actually want to do is skip the test if we see an error right? Which is what CI is going to do since it doesn't have these env variables configured?

Reply:
getAWSKMSURI returns both a correct and incorrect URI, no errors, but I only need the correctURI which is why I'm ignoring the incorrect one. It already skips the test for you if the env variable isn't set, which is why you have to pass in t. Currently already skips the test if not set and seems to be passing Cl.

Only thing getAWSEncryptionOption does is wrap the correctURI in single quotes so I didn't have to do Sprintf and do it every single time.

Reply:
my bad, we have two `getAWSKMSURI` in the codebase, I was looking at the wrong one. Thanks for the explanation, my only remaining comment is that you've run these tests once with actual KMS keys.

Reply:
Yup I've run them with keys and all seems to be passing!

Reply:
Nope, 🚢 it!